site/_scss/blocks/_masonry-grid.scss

```diff
@@ -16,7 +16,6 @@
     display: grid;
     gap: get-size(500);
     grid-template-columns: 1fr 1fr;
-    grid-template-columns: 1fr;
```

Comment:
This was the issue on Windows? Wild.

Reply:
No sorry I misspoke. I'm writing a longer comment